### ISSUE TITLE
Add support for notifications

### ISSUE THREAD

**Issue by EndBug** [toto04/webeep-sync#38]
For those who use them in moodle downloader 2, it would be nice to have a way to send notifications either via webhook, telegram, mail, or else.

**Comment by toto04** [comment]
yes that would be nice to have, I'll think about implementing it for the full release, thank you!
(btw if you'd like to contribute with this yourself you're more than welcome to lol)
